component-tools/src/main/java/org/talend/sdk/component/tools/ComponentValidator.java

```diff
@@ -72,9 +72,13 @@ public class ComponentValidator extends BaseTask {
 
     private final List<ValidationExtension> extensions;
 
-    public ComponentValidator(final Configuration configuration, final File[] classes, final Object log) {
+    private final File sourceRoot;
+
+    public ComponentValidator(final Configuration configuration, final File[] classes, final Object log,
```

Comment:
why not declare 2 constructors? Is it required to have this minor breaking change here?